internal/backends/sqlite/metadata/registry.go

```diff
@@ -169,6 +198,33 @@ func (r *Registry) CollectionCreate(ctx context.Context, dbName string, collecti
 	return true, nil
 }
 
+// CollectionGet returns table name associated with provided collection.
```

Comment:
Why `CollectionGet`? Shouldn't it be something like `GetTableName`? 

Also, the function is not fully documented, e.g. it doesn't document cases when the database doesn't exist.

Reply:
As `Registry` struct provides functions for Database and Collection operators all of the Collection related functions start with "`Collection`": `CollectionList`, `CollectionCreate`. I thought that using `CollectionGet` would match the convention.

Reply:
* `CollectionList` returns a list of collections;
* `CollectionCreate` creates a collection;
* `CollectionDrop` drops collection;
* `CollectionGet`… does not get a collection.